components/Notifications/SubscribedAuthors.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from 'react'
+import { compose, graphql } from 'react-apollo'
+import { possibleAuthorSubscriptions } from './enhancers'
+import {
+  plainButtonRule,
+  A,
+  Interaction,
+  mediaQueries,
+  linkRule,
+  colors
+} from '@project-r/styleguide'
+import { css } from 'glamor'
+import SubscribeCheckbox from './SubscribeCheckbox'
+import withT from '../../lib/withT'
+import { withMembership } from '../Auth/checkRoles'
+import Box from '../Frame/Box'
+import Loader from '../Loader'
+import { Link } from '../../lib/routes'
+
+const styles = {
+  checkboxes: css({
+    margin: '20px 0'
+  }),
+  authorContainer: css({
+    display: 'flex',
+    flexDirection: 'column',
+    [mediaQueries.mUp]: {
+      flexDirection: 'row',
+      justifyContent: 'space-between'
+    }
+  }),
+  author: css({
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 10
+  }),
+  userLink: css({
+    color: colors.text,
+    textDecoration: 'underline',
+    '&:visited': {
+      color: colors.text
+    }
+  }),
+  checkbox: css({
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 16,
+    ' div': {
+      marginRight: 16
+    },
+    [mediaQueries.mUp]: {
+      ' div': {
+        marginLeft: 16
+      }
+    }
+  })
+}
+
+const SubscribedAuthors = ({
+  t,
+  data: { authors, loading, error },
+  isMember
+}) => {
+  if (!authors || !authors.length) return null
+  const [showAll, setShowAll] = useState(false)
+
+  const visibleAuthors =
+    authors && authors.filter(author => author.user.subscribedByMe.active)
+
+  const totalSubs =
+    authors &&
+    authors.filter(author => author.user.subscribedByMe.active).length
+  return (
+    <Loader
+      loading={loading}
+      error={error}
+      render={() => {
+        return (
+          <>
+            {!isMember && (
+              <Box style={{ margin: '10px 0', padding: 15 }}>
+                <Interaction.P>
+                  {t('Notifications/settings/formats/noMembership')}
```

Comment:
Maybe we could reformulate this message to work for formats and authors and move it above both components? (in `components/Notifications/Settings.js`)